web/styling/box-model/borders.md

```diff
@@ -36,15 +38,19 @@ div{
 }
 ```
 
-Or using the shorthand notation, that respects the same order:
+Or using the shorthand notation:
 
 ```css
 div{
-  border: 4px solid red;
+  border: red 4px solid;
 }
 ```
 
-Borders can have different styles, of which the most common is `solid`. Others are `dashed`,`dotted` or `double`.
+You can declare it with any combination of the `width`, `style`, and `color`, as well as in any order.
```

Comment:
what do you think about this? I tried to highlight that the order and combination doesn't matter
```suggestion
You can declare it with **any combination** of `width`, `style`, and `color` in **any order**.
```

Reply:
yeah  it is better